service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -0,0 +1,455 @@
+| |^* .*[id="upgrading-ossm"]
+= Upgrading {ProductName} from version 1.1 to version 2.0
+include::modules/ossm-document-attributes.adoc[]
+:context: upgrading-ossm
+toc::[]
+
+To access the most current features of {ProductName}, upgrade to the most current version, 2.0. Upgrading from version 1.1 to 2.0 requires manual steps that migrate your workloads and apps to a new instance of {ProductName} running the new version.
+
+.Prerequisites
+
+* You must upgrade to {product-title} 4.6. before you upgrade to {ProductName} 2.0.
+* You must have an updated {ProductName} operator. If you selected the *automatic* upgrade path, the operator automatically downloads the latest information. However, there are steps you must take to use the features in {ProductName} to version 2.0.
+
+[id="ossm-upgrading-upgrade_{context}"]
+== Upgrading {ProductName}
+
+To upgrade {ProductName}, you must create a new project for the {ProductName} instance. Then, once it's configured, move your microservice applications and workloads to the new instance.
+
+.Procedure
+
+. Back up your control plane configuration. Switch to the project that contains your `ServiceMeshControlPlane` resource.
++
+[source,terminal]
+----
+oc project istio-system
+----
++
+. Enter the following command to retrieve the current configuration. Your <smcp_name> is specified in the metadata of your `ServiceMeshControlPlane` resource, for example `basic-install` or `full-install`.
++
+[source,terminal]
+----
+$ oc get servicemeshcontrolplanes.v1.maistra.io <smcp_name> -o yaml > <smcp_name>.v1.yaml
+----
++
+. Generate and retrieve a sample v2 `ServiceMeshControlPlane` that contains information about your configuration. You can modify this version of your `ServiceMeshControlPlane` as a starting point.
++
+[source,terminal]
+----
+$ oc get smcp <smcp_name>.v2.yaml
```

Comment:
I don't think this will create an example SMCP. Let me create a cluster to confirm.

Reply:
The operator converts this on upgrades. No samples generated.

Reply:
This should be `oc get smcp <smcp_name>`